website/docs/how-to/how-to-add-sso-open-id-connect.md

```diff
@@ -76,3 +76,45 @@ Log out of Unleash and sign back in again. You should now be presented with the
+| OIDC_DISCOVER_URL  | Tells Unleash how to configure OIDC                                   | yes      | https://myoidchost.azure.com/.well-known/openid-configuration | 
+| OIDC_CLIENT_ID     | The OIDC client ID of this application.                               | yes      | FB87266D-CDDB-4BCF-BB1F-8392FD0EDC1B                          |
+| OIDC_CLIENT_SECRET | Shared secret from OpenID server. Used to authenticate login requests | yes      | qjcVfeFjEfoYAF3AEsX2IMUWYuUzAbXO                              |
+
+Once these are configured OIDC should be working.
+
+### Step 2 (optional) - Auto create users from specific email domains
+
+| Variable name                  | Purpose                                                                                        | Required | Example values              |
+|--------------------------------|------------------------------------------------------------------------------------------------|----------|-----------------------------|
+| OIDC_AUTO_CREATE               | Tells Unleash to auto create users from the specific domains in OIDC_AUTO_CREATE_EMAIL_DOMAINS | no       | true / false                |
+| OIDC_AUTO_CREATE_EMAIL_DOMAINS | A comma separated list of domains to auto-create users for                                     | no       | getunleash.io,getunleash.ai | 
+
+### Step 3 (optional) Enable group syncing
+
+| Variable name               | Purpose                                                                                                  | Required   | Example values              |
+|-----------------------------|----------------------------------------------------------------------------------------------------------|------------|-----------------------------|
+| Variable name               | Purpose                                                                                                  | Required   | Possible values             |
+| --------------------------- | -------------------------------------------------------------------------------------------------------- | ---------- | -----------------           |
+| OIDC_ENABLE_GROUP_SYNCING   | Tell Unleash to setup group syncing from OIDC login requests                                             | No         | true or false               |
+| OIDC_GROUP_JSON_PATH        | a json path expression telling where in the response Unleash can find the group membership information   | No         | groups                      |
+| OIDC_ADD_GROUPS_SCOPE       | Tells Unleash to add the `groups` access scope to the request                                            | No         | true / false                |                                                                    
+| OIDC_DEFAULT_ROOT_ROLE      | Which role to grant users auto created from SSO, defaults to Viewer                                      | No         | 'Viewer', 'Editor', 'Admin' |
+
+### Step 4 (optional) - Further customizations
```

Comment:
I'm not sure I'd call them steps, as you usually configure all env variables in a single step.